test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -13,6 +15,76 @@ public class MsbuildIntegrationTestFixture : IDisposable
     {
         internal readonly string _testDir;
         private readonly Dictionary<string, string> _processEnvVars = new Dictionary<string, string>();
+        private readonly Lazy<string> _msbuildPath = new Lazy<string>(() =>
```

Comment:
Say I have 2 VS instances on my machine, stable & preview. 

Is there a difference in the MSBuild instances discovered when I ran tests in the stable vs preview VS?

Reply:
If you launch VS from the Developer Command Prompt/Powershell, then msbuild.exe for that specific instance will be on the path, and that way you can get the tests to use that specific version of msbuild.